icechunk-python/examples/high_concurrency.py

```diff
@@ -311,11 +331,13 @@ def report(self) -> None:
 def mk_client(
```

Comment:
we should probably set a default `keepalive` and allow it to be configurable too.